server/src/com/cloud/alert/AlertManagerImpl.java

```diff
@@ -767,7 +767,9 @@ public void sendAlert(AlertType alertType, long dataCenterId, Long podId, Long c
                 // set up a new alert
                 AlertVO newAlert = new AlertVO();
                 newAlert.setType(alertType.getType());
-                newAlert.setSubject(subject);
+                //do not have a seperate column for content.
+                //appending the message to the subject for now.
+                newAlert.setSubject(subject+content);
```

Comment:
Are you sure this is a good idea?
If the column for content does exist, what about creating it with this PR? Thus, we can avoid this type of half measure solution. Especially that the column `subject` has a limitation on its size  `@Column(name = "subject", length = 999)`; this can potentially create problems in certain conditions in runtime.

Reply:
agree with @rafaelweingartner